packages/web/README.md

```diff
@@ -245,6 +245,12 @@ module.exports = {
         removeDebugger: true,
       },
 
+      // Remove the contents of the output directory prior to building.
+      // Set to false to disable cleaning this directory
+      clean: {
+        paths: [neutrino.options.output]
```

Comment:
Ah - this addition is to a different part of the file than that in `docs/packages/web/README.md`. I don't suppose you could open a PR to sync them up? (The more things are in sync, the easier it will be when I come to land the docs work)

Reply:
Definitely!

Reply:
Addressed in #544.